### PR TITLE
Create a column to indicate whether a meter exists for a property

### PR DESCRIPTION
**Pull request by anchapin**
**Any background context you want to provide?**

Two things I have questions about are in the "tax_lot_properties.py" and inventory controller files. I use an if statement to check whether it is dealing with a Tax Lot or a Property and then only execute the code for the Property since Tax Lots don't have meters. I'm guessing that there is a better way to do this, especially for the inventory controller files, so suggestions are welcome.

**What's this PR do?**

Adds a column between Notes and link to Detail Page on the "View by Property" tab of both Properties List pages. This will indicate whether a property has any meters. You can click on the lightning bolt icon to go to the Meters page of the property.

**How should this be manually tested?**
Add at least one meter to a property and go back to the Inventory page and look for the dark lightning bolt icon (image).

**What are the relevant tickets?**

#2885

### CODE REVIEW

File: seed/models/tax_lot_properties.py [hunk 1]
```diff
@@ -186,6 +186,11 @@ def serialize(
             state_id__in=models.Subquery(states_qs.values('state_id'))
         ).values_list('state_id', flat=True)
 
+        # gather meter counts
+        # Meter = apps.get_model('seed', 'Meter')
+        # obj_meter_counts = {x[0]: x[1] for x in Meter.objects.filter(**{lookups['obj_query_in']: ids})
+        #                     .values_list(lookups['obj_view_id']).order_by().annotate(Count(lookups['obj_view_id']))}
+
```
Comment (perryr16) [inline seed/models/tax_lot_properties.py:193]: Can we delete this comment?

Comment (anchapin) [reply seed/models/tax_lot_properties.py]: I deleted this comment.

**Review by perryr16 — APPROVED** [review]
Works as described. Are the conflicts with `package-lock.json` and `npm-shrinkwrap.json` expected?

**Comment by anchapin** [comment]
I'm not really sure what happened with `package-lock.json` and `npm-shrinkwrap.json`. I tried to edit them to be the same as the develop branch though.

**Comment by perryr16** [comment]
Sometimes rerunning the tests is all it takes

**Pull request merged**